surfsense_backend/app/config/__init__.py

```
@@ -96,9 +96,18 @@ class Config:
     # OAuth JWT
     SECRET_KEY = os.getenv("SECRET_KEY")
     
-    # Unstructured API Key
-    UNSTRUCTURED_API_KEY = os.getenv("UNSTRUCTURED_API_KEY")
+    # ETL Service
+    ETL_SERVICE = os.getenv("ETL_SERVICE")
     
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)